browse/controllers/abs_page/__init__.py

```diff
@@ -70,7 +68,6 @@ def get_abs_page(arxiv_id: str) -> Response:
     ------
     :class:`.InternalServerError`
         Raised when there was an unexpected problem executing the query.
-
```

Comment:
Sorry to start off nitpicking, but my linter and pydocstyle will complain that this violates PEP257, D413 ([Missing blank line after last section](http://www.pydocstyle.org/en/2.1.1/error_codes.html)).

Reply:
I guess this is not technically a problem with the numpy convention we're using.